messages/proposed_event.go

```diff
@@ -1,14 +1,108 @@
 package messages
 
 import (
+	"encoding/json"
+
 	uuid "github.com/gofrs/uuid"
 )
 
 // ProposedEvent ...
 type ProposedEvent struct {
-	EventID      uuid.UUID
-	EventType    string
-	ContentType  string
-	Data         []byte
-	UserMetadata []byte
+	eventID      uuid.UUID
+	eventType    string
+	contentType  string
+	data         []byte
+	metadata []byte
+}
+
+func NewJsonProposedEvent(eventType string, payload interface{}) (ProposedEvent, error) {
+	event := ProposedEvent {}
```

Comment:
Cosmetics, but it would be nice to run `go fmt` here (or even better: `goimports` :wink:) to format code according to Go standards.

Reply:
it was formatted by `goimports`, maybe there are some settings I should apply?

Reply:
![image](https://user-images.githubusercontent.com/5737091/134960599-76d9ef09-34d0-4620-b344-be3d9761d39b.png)
after running goimports on my side

are you running

```
goimports  -l -w 
```
?

Reply:
Yes I was